Makefile

```diff
@@ -136,6 +117,118 @@ auto_usera.o: \
 compile auto_usera.c
 	./compile auto_usera.c
 
+auto_uida.o: \
+compile auto_uida.c uidgid.h auto_uids.h auto_users.h
+	./compile auto_uida.c
+
+auto_userd.c: \
+auto-str conf-users
+	./auto-str auto_userd `head -2 conf-users` > auto_userd.c
```

Comment:
All the generated `auto_user*.c` are identical (!), getting the first line from `conf-users` (likewise for `auto_group*.c`). In the original sequences, files after the first one had a [`| tail -1`](https://github.com/notqmail/notqmail/pull/15/files#diff-b67911656ef5d18c4ae36cb6741b7965L116), which got lost here. Please re-add.

Reply:
Dang, that was sloppy.  Fixed.